spi/src/main/java/org/eclipse/dataspaceconnector/spi/transfer/TransferProcessManager.java

```diff
@@ -33,4 +35,11 @@
      */
     TransferInitiateResult initiateProviderRequest(DataRequest dataRequest);
 
+    void transitionRequestAck(String processId);
```

Comment:
We will likely substitute this with commands from the CommandQueue later, so can we add a comment indicating that here?

Reply:
Sure. 
I was thinking about the api that the `TransferProcessStore` interface will have in future, do you think a single `execute(command)` method will be enough or better to have a method for each state change require?
Choosing the first option will mean that every implementation of `TransferProcessStore` would need to know about the commands, for me is ok, but is something we have to keep in mind